action.yml

```diff
@@ -6,17 +6,29 @@ inputs:
   diff-stat:
     description: 'The JSON object describing changes. This is the "stat" output of lykahb/paths-filter.'
     required: true
+  minimum-files:
+    description: 'The minimum number of files that must change before a comment is posted'
+    default: 0
+  minimum-lines:
+    description: 'The minimum number of lines of code that must change before a comment is posted'
```

Comment:
Can you add a comment that says if both of those two conditions must satisfy or one is enough?